rites/roman1969/src/calendars/countries/hong-kong/index.ts

```diff
@@ -0,0 +1,81 @@
+import { PatronTitle } from '../../../constants/martyrology-metadata';
+import { Precedences } from '../../../constants/precedences';
+import { CalendarDef } from '../../../models/calendar-def';
+import { Inputs } from '../../../types/calendar-def';
+
+export class HongKong extends CalendarDef {
+  particularConfig = {
+    ascensionOnSunday: true,
+    epiphanyOnSunday: true,
+    corpusChristiOnSunday: true,
+  };
+
+  inputs: Inputs = {
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    mary_mother_of_god: {
+      isHolyDayOfObligation: false,
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    joseph_freinademetz_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 29 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    gabriele_maria_allegra_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 30 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 2, date: 25 },
+      martyrology: ['aloysius_versiglia_bishop', 'callistus_caravario_priest'],
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    our_lady_of_china: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 5, dayOfWeek: 6, nthWeekInMonth: 2 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    our_lady_help_of_christians: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 5, date: 24 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    holy_martyrs_and_blesseds_of_china: {
+      precedence: Precedences.ProperSolemnity_PrincipalPatron_4a,
+      dateDef: { month: 7, date: 9 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: {
+      precedence: Precedences.ProperFeast_8f,
+      dateDef: { month: 10, date: 1 },
+    },
```

Comment:
the elevation isn't specified in the calendar, probably specify
```suggestion
    // src: http://catholic-dlc.org.hk/10-01Saint_The%CC%81re%CC%80se_of_the_Child_Jesus.doc
    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: {
      precedence: Precedences.ProperFeast_8f,
      dateDef: { month: 10, date: 1 },
    },
```
instead?

Reply:
yes, thanks for the suggestion

Reply:
This source states that the celebration is raised to memorial, but only in the Diocese of Hong Kong.

```suggestion
    // src: http://catholic-dlc.org.hk/viewer.htm?doc=10-01Saint_The%CC%81re%CC%80se_of_the_Child_Jesus.doc
    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: {
      precedence: Precedences.ProperFeast_8f,
      dateDef: { month: 10, date: 1 },
    },
```

This is where it occurred to me whether we should consider Hong Kong as part of China or not. I don’t want to politicise this, however, I consider Hong Kong as part of China, even though it is a _special administrative region_. Therefore, we should move the calendar `rites/roman1969/src/calendars/countries/china/hong-kong.ts`. This also means removal of all celebrations which are inscribed into `china/index.ts` and possibly drop celebrations inscribed to `china/index.ts`, but not `hong-kong.ts`. WDYT, @TobiTenno? 🤔

Reply:
This one is not yet implemented as I see it.

Reply:
> This one is not yet implemented as I see it.

I would prefer not to place the Hong Kong calendar under China, as doing so would require overriding or removing most of the celebrations specific to China. There is no clear practical benefit to this, other than adhering to a hierarchical structure that is itself still open to debate, particularly regarding whether Hong Kong should be considered part of China within a religious structure.

I have consulted local priests in Hong Kong, and they have clearly stated that the Diocese of Hong Kong is independent from mainland China, with separate administration and distinct liturgical celebrations.

Source: https://catholic.org.hk/en/hong-kong-catholic-diocese/catholic-diocese-of-hong-kong